src/localconf.js

```diff
@@ -85,6 +85,7 @@ function buildConfig(opts) {
   const coreMetadataPath = `${hostname}coremetadata/`;
   const indexdPath = typeof indexdURL === 'undefined' ? `${hostname}index/` : ensureTrailingSlash(indexdURL);
   const wtsPath = typeof wtsURL === 'undefined' ? `${hostname}wts/oauth2/` : ensureTrailingSlash(wtsURL);
+  const wtsAggregateUserUserPath = `${hostname}wts/aggregate/user/user`;
```

Comment:
probably better to call it `wtsAggregateAuthzPath` (in consistent with the current Arborist `authzPath` and better to understand what is it actually for)